Tests/ProfilerTest.php

```diff
@@ -42,6 +42,8 @@ public function setUp()
 
         $twigLoaderFilesystem = new FilesystemLoader(__DIR__ . '/../Resources/views/Collector');
         $twigLoaderFilesystem->addPath(__DIR__ . '/../vendor/symfony/web-profiler-bundle/Resources/views', 'WebProfiler');
+        // This is normally added by the TwigBundle TwigExtension class
```

Comment:
The problem here is that symfony/twig-bundle will only be a true dependency of symfony/web-profiler-bundle in the next 3.4 (and all upper branches) release 😕 Should we bump min required version of WPB or ignore it?

Reply:
That's fine to me:

- this test does not rely on TwigBundle as it configures Twig by itself
- projects using WebProfilerBundle will already need TwigBundle anyway to configure twig. That's already the case since SF 2.0.